modules/openapi-generator/src/main/resources/htmlDocs2/index.mustache

```diff
@@ -111,10 +111,10 @@
         } else {
             // Use a for loop instead of forEach to avoid nested functions
             // Otherwise "return" will not work properly
-            for(var propt in currentNode){
-                if (currentNode.hasOwnProperty(propt)) {
-                    currentChild = currentNode[propt]
-                    if (id == propt) {
+            for(var prompt in currentNode){
+                if (currentNode.hasOwnProperty(prompt)) {
+                    currentChild = currentNode[prompt]
+                    if (id == prompt) {
```

Comment:
i think this should be `property` instead of prompt/propt as it stands for property

i will take care of it in another PR